docs/projects/adrv9361z7035/index.rst

```diff
@@ -202,20 +202,21 @@ command.
    ============= ===============================================
    ccbob_cmos    ADRV9361Z7035-SOM (CMOS Mode) + ADRV1CRR-BOB
    ccbob_lvds    ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-BOB
-   ccpackrf_lvds ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-PACKRF
+   ccpackrf_lvds ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-PACKRF*
    ccfmc_lvds    ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-FMC
    ccpci_lvds    ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-PCI
    ccusb_lvds    ADRV9361Z7035-SOM (LVDS Mode) + ADRV1CRR-USB
    ============= ===============================================
-   
+
+   * removed, last release that supported the project https://github.com/analogdevicesinc/hdl/tree/hdl_2021_r2
```

Comment:
Create an admonition legend with class note, just like the legend from down below... and write the link in the :git-hdl: format

Reply:
done